### PR TITLE
Upgrade to node 10

### PR DESCRIPTION
**Pull request by regiskuckaertz**
This one I can actually test on CODE ✌️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,19 +5,22 @@
...
   "buildDir": "./target",
   "riffraffFile": "./riff-raff.yaml",
+  "engines": {
+    "node": ">=10.0.0"
+  },
   "dependencies": {
     "aws-sdk": "^2.385.0",
-    "dotenv": "^6.2.0",
-    "node-fetch": "^2.2.0"
+    "dotenv": "8.2.0",
```
Comment (JustinPinner) [inline package.json]: Was it intentional to drop the leading `^` here?

File: package.json [hunk 2]
```diff
@@ -5,19 +5,22 @@
...
+    "node-fetch": "^2.6.0"
   },
   "devDependencies": {
-    "@types/node": "^10.12.1",
-    "node-riffraff-artefact": "^2.0.1",
-    "prettier": "^1.14.3",
-    "tslint": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
+    "@types/node": "12.12.7",
```
Comment (JustinPinner) [inline package.json]: And here (`^`)

File: package.json (hunk 1 above)
Comment (regiskuckaertz) [reply package.json]: I don't know, I just used `yarn add` 🤔 I can put it back.

Comment (JustinPinner) [reply package.json]: ¯\_(ツ)_/¯ It's probably not critical

**Pull request merged**